scopes/toolbox/regex/node-modules-exclude-packages/node-modules-exclude-packages.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { nodeModulesExcludePackages } from './node-modules-exclude-packages';
+
+describe('nodeModulesExcludePackages()', () => {
+  it('should return an empty regex when the received array is empty', () => {
+    expect(nodeModulesExcludePackages({ packages: [] })).toEqual('node_modules/(?!()/)');
+  });
+  it('should return a regex when the received array is not empty', () => {
+    const packagesToTransform = [
+      'lit',
+      '@lit',
+      'testing-library__dom',
+      '@open-wc',
+      'lit-html',
+      'lit-element',
+      'pure-lit',
+      'lit-element-state-decoupler',
```

Comment:
you don't need to add such a long list in a test.
enough to use 2 packages

Reply:
ok, I'll update it.